demo/common/assets.js

```diff
@@ -653,28 +653,37 @@ shakaAssets.testAssets = [
       .addFeature(shakaAssets.Feature.WEBM)
       .addFeature(shakaAssets.Feature.OFFLINE),
   new ShakaDemoAssetInfo(
-      /* name= */ 'Shaka Player History (multicodec, live, DASH)',
+      /* name= */ 'Shaka Player History (H264, VP9, AV1, live, DASH)',
       /* iconUri= */ 'https://storage.googleapis.com/shaka-asset-icons/shaka.png',
       /* manifestUri= */ 'https://storage.googleapis.com/shaka-live-assets/player-source.mpd',
       /* source= */ shakaAssets.Source.SHAKA)
-      .markAsDisabled()
+      .addDescription('A self-indulgent DASH live stream.')
+      .markAsFeatured('Shaka Player History: Live')
       .addFeature(shakaAssets.Feature.DASH)
       .addFeature(shakaAssets.Feature.HIGH_DEFINITION)
       .addFeature(shakaAssets.Feature.LIVE)
       .addFeature(shakaAssets.Feature.MP4)
       .addFeature(shakaAssets.Feature.WEBM),
   new ShakaDemoAssetInfo(
-      /* name= */ 'Shaka Player History (live, HLS)',
+      /* name= */ 'Shaka Player History (H264, VP9, AV1, live, HLS)',
       /* iconUri= */ 'https://storage.googleapis.com/shaka-asset-icons/shaka.png',
       /* manifestUri= */ 'https://storage.googleapis.com/shaka-live-assets/player-source.m3u8',
       /* source= */ shakaAssets.Source.SHAKA)
-      .addDescription('A self-indulgent HLS livestream.')
-      .markAsFeatured('Shaka Player History')
-      .markAsDisabled()
+      .addDescription('A self-indulgent HLS live stream.')
       .addFeature(shakaAssets.Feature.HIGH_DEFINITION)
       .addFeature(shakaAssets.Feature.HLS)
       .addFeature(shakaAssets.Feature.LIVE)
-      .addFeature(shakaAssets.Feature.MP4),
+      .addFeature(shakaAssets.Feature.MP4)
+      .setExtraConfig({
+        manifest: {
+          // HLS doesn't signal the availability window, but this stream is
+          // seekable.
+          availabilityWindowOverride: 3600,
+          // This is the presentation delay signaled explicitly in the DASH
+          // version.
+          defaultPresentationDelay: 60,
```

Comment:
I understand that you want this so that both streams have the same latency, but it's not really necessary, why should we use this?